src/libs/actions/Report.ts

```diff
@@ -787,9 +787,7 @@ function openReport(
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
             value: {
-                errorFields: {
-                    notFound: null,
-                },
```

Comment:
I think you commit this change by false.

Reply:
Yeah, sorry I should add a comment on the PR, this change is because of this discussion: https://callstack-hq.slack.com/archives/C05LX9D6E07/p1717509334788509?thread_ts=1717508731.782149&cid=C05LX9D6E07

Reply:
I didn't have access to this channel. Is there is any test steps for it?

Reply:
No, it was a bug that happen to David, and with @mountiny didn't know well how it happens. But it seems that not all the errors were cleaned up successfully.

Reply:
@gedu this change seems wrong and is incorrectly clearing all report errors. I'm intending on reverting it. I'm not sure which issue you were trying to fix with this change (I don't have access to that slack conversation either) but this is probably not the right way to fix it.